yb-voyager/src/callhome/diagnostics.go

```diff
@@ -33,35 +32,122 @@ import (
 
 // call-home json formats
 var (
-	jsonFilePath    string
-	Payload         payload
 	SendDiagnostics utils.BoolStr
 )
 
-const (
+var (
 	CALL_HOME_SERVICE_HOST = "34.105.39.253"
 	CALL_HOME_SERVICE_PORT = 80
 )
 
-type payload struct {
-	MigrationUuid         uuid.UUID `json:"UUID"`
-	StartTime             string    `json:"start_time"`
-	YBVoyagerVersion      string    `json:"yb_voyager_version"`
-	LastUpdatedTime       string    `json:"last_updated_time"`
-	SourceDBType          string    `json:"source_db_type"`
-	SourceDBVersion       string    `json:"source_db_version"`
-	Issues                string    `json:"issues"`
-	DBObjects             string    `json:"database_objects"`
-	TargetDBVersion       string    `json:"target_db_version"`
-	NodeCount             int       `json:"node_count"`
-	ParallelJobs          int       `json:"parallel_jobs"`
-	TotalRows             int64     `json:"total_rows"`
-	TotalSize             int64     `json:"total_size"`
-	LargestTableRows      int64     `json:"largest_table_rows"`
-	LargestTableSize      int64     `json:"largest_table_size"`
-	TargetClusterLocation string    `json:"target_cluster_location"` //TODO
-	TargetDBCores         int       `json:"target_db_cores"`         //TODO
-	SourceCloudDBType     string    `json:"source_cloud_type"`       //TODO
+/*
+Call-home diagnostics table structure -
+CREATE TABLE diagnostics (
+
+	migration_uuid UUID,
+	phase_start_time TIMESTAMP WITH TIME ZONE,
+	source_db_details JSONB,
+	target_db_details JSONB,
+	yb_voyager_version TEXT,
+	migration_phase TEXT,
+	phase_payload JSONB,
+	migration_type TEXT,
+	time_taken_sec NUMERIC(20,2),
+	status TEXT,
+	PRIMARY KEY (migration_uuid, phase_start_time, migration_phase)
```

Comment:
For live migration /long running processes, where we aim to send multiple rows for a command, this won't suffice, correct? 

I think it might be worth thinking about this right now, even if don't implement live migration. That way, the schema will be fixed, and we just have to enhance voyager later on..

Reply:
> For live migration /long running processes, where we aim to send multiple rows for a command, this won't suffice, correct?

so for this what I was thinking is that when I will send the payload for a log running command I will modify the start time to the time just after the previous payload was sent and accordingly update the timeTaken. So that way this PK would suffice. What do you say?

Reply:
I think that might be a bit confusing. (it might seem like the user stop/start command multiple times). I think we should have a clearer way of representing this.
Maybe we can add another timestamp field - collected_at, which would be different for paylaods sent at different times. but then time_taken does not make sense for these type of payloads...

We can discuss this once in the stand-up 👍

Reply:
Made this change in #1555